llvm/test/FileCheck/.gitattributes

```diff
@@ -0,0 +1 @@
+dos-style-eol.txt
```

Comment:
missing text=auto eol=crlf ?

Reply:
Thanks `text eol=crlf`. Updated in 64350b342a09ba69803a541a89b5681a12925ff0